.husky/pre-commit

```diff
@@ -1,4 +1,8 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
+pnpm run i18n:extract
+git add apps/saleor-app-checkout/content/locales
+git add packages/checkout-storefront/content/locales
```

Comment:
hmm... I'm not a big fan of running long-running commands and adding new files in commits automatically - could we instead check if the locales were extracted on `git push`?

IMHO, we should offload checks as much as possible to CI, so we don't have to wait locally for stuff to complete - this is a huge issue in dashboard, every commit takes ~30 seconds + 2/3 minutes on each push because of slow commands. This is really irritating for small changes

Reply:
`check if the locales were extracted` - this would take the same amount of time or even more, and when it fails you'd need to run the command and `git add` manually, and try to commit again. I don't see a point in that if we can automate.

Reply:
But we're not running TS or Eslint on commit @bmigirl so I'm not sure what are you referring to

Reply:
And you really shouldn't commit with `no-verify` :P

Reply:
https://github.com/saleor/react-storefront/issues/538